### PR TITLE
fix(3163900810): align discord button to Figma

### PR DESCRIPTION
**Pull request by galuzan-starkware**
### Description of the Changes

- Changing the size of the button
- Changing padding and border radius

Solves #[3163900810](https://starkware.monday.com/boards/2891230138/pulses/3163901936)

---

### Checklist

- [x] Manually tests of the main Application flows are done and passed.
- [ ] New unit / functional tests have been added (whenever applicable).
- [ ] Docs have been updated (whenever relevant).
- [x] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/305)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const App = () => {
           <Route element={<Navigate replace to="/" />} path="*" />
         </Routes>
       </StyledBackground>
-      <SideButton icon={<DiscordIcon />} onClick={onDiscordClick} />
+      <SideButton icon={<DiscordIcon height={98} width={26} />} onClick={onDiscordClick} />
```
Comment (dan-ziv) [inline src/App.js:44]: Change it in the `svg` itself.

Comment (galuzan-starkware) [reply src/App.js]: fixed

**Review by dan-ziv — APPROVED** [review]

**Pull request merged**